.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 work
+/github-branch-source.iml
```

Comment:
Suggesting install gitignore plugin if you are using IDEA or manually copy-paste IDEA, maven, java, eclipse, gradle (for future) `ignores` from https://github.com/github/gitignore